### DISCUSSION TITLE
Change the colour of recommendation text

### DISCUSSION THREAD

**Discussion by ckm66** [ohmyzsh/ohmyzsh#11559] [Help] [answered]
Before the latest update, the colour of the recommendation used to be blue. However, after the update, it changed to blue. I tried to change it by switching themes and the terminal setting but failed. Could I know how I could change the colour of it? Thank you so much
<img width="1265" alt="Screenshot 2023-03-13 at 12 30 09 AM" src="https://user-images.githubusercontent.com/99936596/224558405-48c6ba20-c719-4070-8aca-8a813c403359.png">

**Comment by carlosala** [comment]
@mcornella can you check pls?

**Comment by mcornella** [comment, marked as answer]
The current behavior is supposed to match the color of `ls`. Is that the same or not?

If not, you can use the code below in your zshrc file
```zsh
zstyle ':completion:*' list-colors ${(s.:.)LS_COLORS} 
```
and use https://geoff.greer.fm/lscolors/ to pick the colors you want
